locales-pending/plus-expiration.ftl

```diff
@@ -0,0 +1,37 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+plus-expiration-header-logo-alt = { -brand-mozilla-monitor }
+plus-expiration-intro-heading = Renew your subscription and <b>get 10% off</b>
+plus-expiration-intro-content = Stay protected with data breach alerts and info removals – renew your subscription today.
+plus-expiration-intro-cta-label = Renew subscription
+plus-expiration-intro-terms = Limited terms and restrictions apply
+plus-expiration-intro-generic-error = Couldn’t renew subscription. <retry-button>Try again</retry-button>
+plus-expiration-intro-generic-error-dismiss = Dismiss
+
+plus-expiration-confirm-heading = Thanks for renewing your subscription
+plus-expiration-confirm-content = Go to your dashboard to see the latest on how we’re protecting your data.
+plus-expiration-confirm-cta-label = Go to { -brand-monitor }
+plus-expiration-confirm-manage-button = Manage subscription
+
+plus-expiration-error-not-expired-heading = Your { -brand-monitor-plus } subscription is still active
+plus-expiration-error-not-expired-content = Go to your dashboard to see the latest on how we’re protecting your data.
+plus-expiration-error-not-expired-cta-label = Go to { -brand-monitor }
+
+plus-expiration-error-free-heading = Discount valid for current { -brand-monitor-plus } customers
+plus-expiration-error-free-content = Subscribe to { -brand-monitor-plus } to get protected with data breach alerts and info removals.
+plus-expiration-error-free-cta-label = Subscribe to { -brand-monitor-plus }
+
+plus-expiration-error-already-applied-heading = Discount code already applied
+plus-expiration-error-already-applied-content-part1 = Looks like this discount code was used for a subscription to this account.
+plus-expiration-error-already-applied-content-part2 = <support-link>Contact our Support team</support-link> for help or head to your dashboard to see the latest on how we’re protecting your data.
+plus-expiration-error-already-applied-cta-label = Go to { -brand-monitor }
+
+plus-expiration-terms-heading = <b>Terms and restrictions</b> 10% off subscription renewal
```

Comment:
Is it just me or this string seems a bit strange? How does it look in the website?

Reply:
Heh, now that you mention it it does, but it looked fine initially, and I think it's technically correct (though maybe needs a dash somewhere) - it's the terms for getting 10% off of the renewal of your subscription.

Here's what it looks like:

![image](https://github.com/user-attachments/assets/60dd870c-04d2-4a38-afe6-20ed42da2800)

Reply:
I'll check with content.

Reply:
OK, they're on separate lines now, and therefore, also separate strings:

![image](https://github.com/user-attachments/assets/0b1a67e6-f555-4fe0-a094-bd5c08082b54)